### PR TITLE
N17: Building Pikachu Volleyball

### PR DESCRIPTION
**Pull request by danbugs**
Part of #408

### CODE REVIEW

File: content/posts/newsletter-017/index.md [hunk 1]
```diff
@@ -87,6 +87,30 @@ This month's updates:
...
+Here, [@danlogs] re-creates the Pikachu spin-off game from 1997 by Satoshi Takenouchi
+— Pikachu Volleyball.
+
+Apart from being a good way for aspiring game developers to start
+understanding the gist of game dev with Rust, this video also displays some of
+the language's hardest topics (i.e. ownership, lifetimes, modules, etc.)
+in a real-world application.
+
+_Discussions:
+[danlogs' Discord](https://discord.com/invite/fSWE49H),and
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-017/index.md:106]: Minor formatting/grammar tweak:
```suggestion
[danlogs' Discord](https://discord.com/invite/fSWE49H) and
```

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/posts/newsletter-017/index.md (hunk 1 above)
Comment (danbugs) [reply content/posts/newsletter-017/index.md]: Done ✅

**Review by ozkriff — APPROVED** [review]
:+1:  (made a few style & fmt tweaks for consistency with the rest of the newsletter)

**Comment by ozkriff** [comment]
Thanks!

**Pull request merged**